src/css/main.scss

```diff
@@ -0,0 +1,226 @@
+// Colors
+$highlight: #DB392E;
+$structure: #77C8E5;
+$background: white;
+$subtle: #FDF8EE;
+$text: black;
+$light: #DAE1E1;
+
+// Bootstrap Overrides
+$brand-primary: $highlight;
+$font-family-base: -apple-system, BlinkMacSystemFont,
+  "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell",
+  "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+$body-bg: $background;
+$body-color: $text;
+$link-color: $highlight;
+$headings-color: $structure;
+$input-border-focus: $highlight;
+$input-border-color: $light;
+$input-color-placeholder: $light;
+$btn-secondary-border: $light;
+
+// Bootstrap
+@import "../dependencies/bootstrap/scss/bootstrap-flex.scss";
```

Comment:
Nils, I don't like relative path here. The problem is that during RPM build we relocate stuff, see
https://github.com/cms-sw/cmsdist/blob/comp_gcc493/wmarchive.spec#L50

and it may affect the import here. Please have a look at vocms013:/data/srv/HG1512e/sw.valya/slc6_amd64_gcc493/cms/wmarchive/v00.00.70
WMArchive dir structure once RPM are in place. It is different from package one. And, make adjustment to import here to make it work properly within a package and RPM build/layout.